Examples/Algorithms/Geant4DD4hep/src/DD4hepDetectorConstruction.cpp

```diff
@@ -25,7 +26,7 @@ DD4hepDetectorConstruction::DD4hepDetectorConstruction(
 G4VPhysicalVolume* DD4hepDetectorConstruction::Construct() {
   dd4hep::sim::Geant4Mapping& g4map = dd4hep::sim::Geant4Mapping::instance();
   dd4hep::DetElement world = m_detector.world();
-  dd4hep::sim::Geant4Converter conv(m_detector, dd4hep::INFO);
+  dd4hep::sim::Geant4Converter conv(m_detector, dd4hep::PrintLevel::VERBOSE);
```

Comment:
Since the log level would have already been defined globally in the DD4hepGeometryService could'nt you just use : `dd4hep::sim::Geant4Converter conv(m_detector);` ?

Reply:
These two settings actually do two different things, because DD4hep logging [does not follow the same logic as our logging](https://github.com/acts-project/acts/issues/730#issuecomment-787727102):

- The log level that is being set here in the `DD4hepDetectorConstruction` dictates at which log level messages related to the `Geant4Converter` will be emitted inside of DD4hep.
- The log level that is being set in the `DD4hepGeometryService` dictates what is DD4hep's logging threshold, that is, what is the minimal log level that a message must have for DD4hep to print it.

Reply:
I reread #730 afterward and understood after asking this. Sorry and thanks for explaining again.